pkg/serverless/logs/logs.go

```diff
@@ -130,8 +121,12 @@ func (l *logMessage) UnmarshalJSON(data []byte) error {
 	// the rest
 
 	switch typ {
-	case logTypePlatformLogsSubscription, logTypePlatformExtension:
-		l.logType = typ
+	case logTypePlatformLogsDropped:
+		var reason string
+		if record, ok := j["record"].(map[string]interface{}); ok {
+			reason = record["reason"].(string)
+		}
+		log.Debugf("Logs were dropped by the AWS Lambda Logs API: %s", reason)
```

Comment:
Instead of logging further below in `processMessage`, logging here makes more sense since all we want to do is log and do nothing else with the message after that.